sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/aio/client.py

```diff
@@ -549,12 +526,11 @@ def list_deleted_certificates(
         deletion-specific information. This operation requires the certificates/get/list
         permission. This operation can only be enabled on soft-delete enabled vaults.
 
-        :param include_pending: Specifies whether to include certificates which are not
-        completely provisioned.
-        :type include_pending: bool
+        :param bool include_pending: Specifies whether to include certificates which are not
+            completely provisioned.
         :return: An iterator like instance of DeletedCertificate
         :rtype:
-         typing.Generator[~azure.keyvault.certificates.models.DeletedCertificate]
+         ItemPaged[~azure.keyvault.certificates.models.DeletedCertificate]
```

Comment:
To ensure this renders as a link to `ItemPaged`, you may need to fully qualify it: `~azure.core.paging.ItemPaged`